internal/services/notification/runevents.go

```diff
@@ -100,8 +101,17 @@ func (n *NotificationService) runEventsHandler(ctx context.Context) error {
 			// this is just a basic handling. Improve it to store received events and
 			// their status in the db so we can also do more logic like retrying and handle
 			// multiple kind of notifications (email etc...)
-			if err := n.updateCommitStatus(ctx, ev); err != nil {
-				n.log.Info().Msgf("failed to update commit status: %v", err)
+			if ev.EventType == types.RunPhaseChanged {
+				if err := n.updateCommitStatus(ctx, ev); err != nil {
+					n.log.Info().Msgf("failed to update commit status: %v", err)
+				}
+				if n.c.WebExposedURL != "" {
```

Comment:
Why this check?

Reply:
@sgotti the check I would like to use is WebhookUrl, WebExposedURL is wrong. So if WebhookUrl is not defined the notification service not send the webhook. Or do you like better I remove the check?

Reply:
Ok it was a typo. The check on WebhookUrl is instead needed.